src/utilities/getNegativeNumbers/getNegativeNumbers.test.js

```diff
@@ -0,0 +1,37 @@
+import { getNegativeNumbers } from './getNegativeNumbers';
+
+describe('The getNegativeNumbers function', () => {
+  describe('when provided with an array', () => {
+    describe('when array contains negative numbers', () => {
+      it('should return an array with only negative numbers', () => {
+        const result = getNegativeNumbers([1, -5, -3, 12, -152]);
+        expect(result).toEqual([-5, -3, -152]);
```

Comment:
Split it into two separate `describe` cases, one containing values that are not numbers